src/main/resources/aws/single-node.yaml

```diff
@@ -62,7 +66,7 @@ Resources:
         -
           DeviceName: /dev/sda1
           Ebs:
-            VolumeSize: 200
```

Comment:
It was 200 before, and the commit changes it to 100. Volume size affects IOPS. The instance can be slower for a smaller drive. Can you explain the change?

Reply:
The change adds a common abstraction over both DC and Server volumes. It didn't exist before, so we're translating inconsistency into consistency. This change is fundamentally incompatible.

### Parametrize elsewhere
We can avoid this forced consistency by moving the parametrization out of `Computer` and into `DataCenterFormula.Builder` and `StandaloneFormula.Builder`.
This does look like it belongs to `Computer`, but we can clean up that design during the next major version.

### Override
We override the `Volume` in `StandaloneFormula` to 200 GB.
However, it will ignore the user-supplied value.
We can override only if the user-supplied value is the default, ie. 100 GB. But what if the user wanted to actually use 100 GB explicitly? We can use a dirty trick. We make the default a very specific value, e.g 99.71 GB and detect that and hope that no user would specify this special value.

Reply:
There's another option, we can accept the forced consistency. Instead of demoting 200 GB to 100 GB, we can bump 100 GB DC to 200 GB DC

100 GB of GP2 EBS costs 1,5 US cents an hour. For a 1-hour test on a 2-node cluster that's 3 cents. Sounds acceptable, especially since there's a workaround.
There might be another consequence in form of longer provisioning queues, but I have not seen EBS as the scarce resource yet.